### ISSUE TITLE
os.showFolderDialog is not showing up on Win10

### ISSUE THREAD

**Issue by Dimonka2** [neutralinojs/neutralino.js#66]
Hi, just got an issue with the new version of Neutrlinojs (4.6.0) that showFolderDialog is not showing on Wi10 at all in case default path is incorrect. An example of stopped working code:
```js
const path = await Neutralino.os.showFolderDialog( 'Select R folder',
        { defaultPath: 'C:/Program Files/R' })
```
I guess even the default folder is incorrect dialog should be shown.
Thanks!

**Comment by kodaloid** [comment]
Hi there. I've tested this, it appears the reason it does not work for you is because you are using forward slashes in the path instead of the Windows specific backslashes. Try this instead, and it should work:

```js
const path = await Neutralino.os.showFolderDialog( 'Select R folder', 
        { defaultPath: 'C:\\Program Files\\R' });
````

We don't currently have this noted in the documentation, so it maybe worth a PR to add this to help others in the future.

Koda

**Comment by shalithasuranga** [comment]
Fixed via #1229. Changes will be released with the next version. Thanks all :tada:
